### ISSUE TITLE
Alpine 5.x images are missing

### ISSUE THREAD

**Issue by pentago** [zabbix/zabbix-docker#770]
What's the hold up with 5.x Alpine images?
Docker hub wasn't updated for 10 months, where do I get latest alpine image?

**Comment by dotneft** [comment]
Are you sure? All images are updated in time. What image are you talking?

**Comment by pentago** [comment]
Yeah, there are no 5.2 alpine images on Docker Hub: https://i.imgur.com/37dhXZ2.png

**Comment by pentago** [comment]
Amazing job.
